buildSrc/src/main/java/Dependencies.kt

```diff
@@ -135,6 +137,8 @@ object Dependencies {
 
     const val mozilla_fxa = "org.mozilla.appservices:fxaclient:${Versions.mozilla_appservices}"
 
+    const val mozilla_nimbus = "org.mozilla.appservices:full-megazord:${Versions.mozilla_appservices}"
```

Comment:
Just leaving a note here similar to @Dexterp37 that this reference would also need to change so we don't need to use the full-megazord.

Reply:
@mhammond does this affect how nimbus is included in the appservices megazord?

Reply:
I don't quite understand the question, but it looks fine to me. Obviously this will only work for versions where the megazord includes nimbus, which as of today isn't master, but I think you already know that.

Reply:
@mhammond to clarify my original comment here, I would have expected the dependency to be a separate nimbus component and not the full megazord as we have for the other app services components.
```suggestion
    const val mozilla_nimbus = "org.mozilla.appservices:nimbus:${Versions.mozilla_appservices}"
```

Reply:
Ah yes, this is a slightly weird artifact of how the nimbus component is currently built, that it doesn't (yet) have a separate component but is instead provided directly by the megazord.

Reply:
Cross-linking https://github.com/mozilla/application-services/pull/3701, which once it's released will provide a `org.mozilla.appservices:nimbus` package.